RELEASE.md

```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
 
 -   [ ] 8.  Test
 
-    Test the entire release.
+    -   Validate that your feature is working according to the requirements
```

Comment:
Validate that the features present in changelog generated during the circleci pipeline work according to the requirements.

Reply:
done